### DISCUSSION TITLE
Missing ssh in docker image

### DISCUSSION THREAD

**Discussion by tgauweiler** [opentofu/opentofu#1029] [General]
Hi,

while trying out opentofu as replacement for terraform we noticed that the opentofu Docker image doesn't have `ssh` installed, [see here](https://github.com/opentofu/opentofu/blob/v1.6.0-beta5/Dockerfile#L8).

The terraform image, [here](https://github.com/hashicorp/terraform/blob/v1.6.6/Dockerfile#L16), has `ssh` installed.

We need `ssh` as we pull terraform modules with ssh [tf docs](https://developer.hashicorp.com/terraform/language/modules/sources#github), so for now we will just build our own Docker image with `ssh` included.

But as opentofu aims to be a `drop-in replacement for Terraform`, please consider adding `ssh` to the docker image.

**Comment by ghost** [comment]
Hi @tgauweiler I've opened a issue #1031 for this.
